### ISSUE TITLE
Override cluster capabilities' pspEnabled if annotation present

### ISSUE THREAD

**Issue by ChrisHaPunkt** [rancher/rancher#21039]
**What kind of request is this (question/bug/enhancement/feature request):**
BUG

**Steps to reproduce (least amount of steps as possible):**
1. Import a rke cluster with activated PSP in rancher 2.2.4
2. Create a new Project/ Edit an existing Project

**Result:**

See no Dropdown for selecting PodSecurityPolicies
![image](https://user-images.githubusercontent.com/4389395/59903922-8a247000-9402-11e9-9463-07e4d26ff583.png)

**Other details that may be helpful:**

In the API view the setting is present and adjustable though.
![image](https://user-images.githubusercontent.com/4389395/59903976-ac1df280-9402-11e9-9a5f-164d9e63a90c.png)

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
- Installation option (single install/HA): HA / 2.2.4

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Imported
- Machine type (cloud/VM/metal) and specifications (CPU/memory): vvm 32GB / 8Cores
- Kubernetes version (use `kubectl version`): v1.13.5

```
(paste the output here)
```

- Docker version (use `docker version`):
```
18.06.3
```

**Comment by deniseschannon** [comment]
After internal discussion, we do not want to blindly show adding PSPs on projects unless we can specifically detect that an RKE cluster has PSP enabled on the cluster admission controller.

**Comment by jiaqiluo** [comment]
The bug is reproduced in  v2.3.1

Steps:
- use RKE to provision a cluster with PSP is enabled 

the snippet of the `rancher-cluater.yml` file
```
services:
  kube-api:
    pod_security_policy: true
```
- run Rancher v2.3.1 single install 
- add the cluster as an imported cluster into Rancher
- add a project
- edit the project, notice that there is no option for PSP
- view the project in API, notice the action `setpodsecuritypolicytemplate` is available, but somehow I cannot send the request from the UI (Error: parser error), but copy and execute from terminal succeeds
- refresh the project API page, see the new value is applied

**Comment by jiaqiluo** [comment]
In Rancher v2.3-head 1b51a5911 with ui index `latest-2.3`, the new annotation `capabilities.cattle.io/pspEnabled= true` cannot be set via the UI.

there will be a new PR to change the name of the annotation.

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher:v2.3-head da8cbdb with the default UI-index `latest-2.3`

Steps:
- use RKE to provision a cluster with PSP is enabled 

the snippet of the `rancher-cluater.yml` file
```
services:
  kube-api:
    pod_security_policy: true
```
- run Rancher v2.3.1 single install 
- add the cluster as an imported cluster into Rancher
- add a project
- edit the project, notice that there is no option for PSP
- view the project in API, notice the action `setpodsecuritypolicytemplate` is available, but sending this action returns  422 error 
- now edit the cluster to add the annotation: **capabilities/pspEnabled= true**
- view the cluster in API, see `"pspEnabled": true` under the `capabilities`
- if adding/editing project, the option for psp shows up
- edit the existing project to set psp to be restricted 
- deploy a workload with privilege 
- the workload is failed to be deployed
<img width="1316" alt="Screen Shot 2019-11-04 at 12 07 33 PM" src="https://user-images.githubusercontent.com/6218999/68150641-679c7a80-fefd-11e9-9520-7c84d55df211.png">

**Comment by jiaqiluo** [comment]
For an imported RKE cluster which does NOT enable `pod_security_policy`, if the annotation is set at the cluster level, then the psp option will show up at the project create/edit page, but the psp will not take effect.
